lib/signalfx/tracing.rb

```diff
@@ -2,12 +2,14 @@
 require 'signalfx/tracing/http_sender'
 require 'signalfx/tracing/register'
 require 'signalfx/tracing/compat'
+require 'signalfx/tracing/sfx_logger'
 require 'thread'
 
 module SignalFx
   module Tracing
     module Instrumenter
 
+      @logger = Logging.logger
```

Comment:
Due to the potential noisiness of the logger we'll likely want to use it lazily: https://github.com/signalfx/signalfx-ruby-tracing/issues/31

Reply:
Fixed.